test/cypress/integration/settings/allow_apple_pay_spec.js

```diff
@@ -2,46 +2,86 @@ describe('Apple Pay', () => {
-    cy.setEncryptedCookies(userExternalId, gatewayAccountId)
+    Cypress.Cookies.preserveOnce('session', 'gateway_account')
+  })
 
-    cy.task('setupStubs', [
-      {
-        name: 'getUserSuccess',
-        opts: {
-          external_id: userExternalId,
-          service_roles: [{
-            service: {
-              gateway_account_ids: [gatewayAccountId],
-              name: serviceName
-            }
+  describe('is disabled', () => {
+    beforeEach(() => {
+      cy.task('setupStubs', [
+        {
+          name: 'getUserSuccess',
+          opts: {
+            external_id: userExternalId,
+            service_roles: [{
+              service: {
+                gateway_account_ids: [gatewayAccountId],
+                name: serviceName
+              }
+            }]
+          }
+        },
+        {
+          name: 'getGatewayAccountSuccessRepeatNTimes',
+          opts: [{
+            gateway_account_id: gatewayAccountId,
+            payment_provider: paymentProvider
+          }, {
+            gateway_account_id: gatewayAccountId,
+            payment_provider: paymentProvider
           }]
         }
-      },
-      {
-        name: 'getGatewayAccountSuccessRepeatNTimes',
-        opts: [{
-          gateway_account_id: gatewayAccountId,
-          payment_provider: 'worldpay'
-        }, {
-          gateway_account_id: gatewayAccountId,
-          payment_provider: 'worldpay',
-          allow_apple_pay: true
-        }]
-      }
-    ])
-  })
+      ])
+    })
 
-  describe('Enable Apple Pay', () => {
-    it('should enable apple pay', () => {
+    it('should show it is disabled', () => {
+      cy.setEncryptedCookies(userExternalId, gatewayAccountId)
       cy.visit('/digital-wallet')
-      cy.title().should('eq', `Manage digital wallet - ${serviceName} worldpay test - GOV.UK Pay`)
-      cy.get('td').contains('Apple Pay').siblings().find('a').contains('Enable').click()
-      cy.get('button').contains('turn on Apple Pay').click()
+      cy.title().should('eq', `Manage digital wallet - ${serviceName} ${paymentProvider} test - GOV.UK Pay`)
+      cy.get('td').contains('Apple Pay').siblings().find('a').contains('Change').click()
+      cy.get('input[type="radio"]').should('have.length', 2)
+      cy.get('input[value="on"]').should('not.be.checked')
+      cy.get('input[value="off"]').should('be.checked')
+    })
+  })
+
+  describe('but allow us to enable', () => {
+    beforeEach(() => {
+      cy.task('setupStubs', [
+        {
+          name: 'getUserSuccess',
+          opts: {
+            external_id: userExternalId,
+            service_roles: [{
+              service: {
+                gateway_account_ids: [gatewayAccountId],
+                name: serviceName
+              }
+            }]
+          }
+        },
+        {
+          name: 'getGatewayAccountSuccessRepeatNTimes',
+          opts: [{
+            gateway_account_id: gatewayAccountId,
+            payment_provider: paymentProvider
```

Comment:
Would it be good to specify that `allow_apple_pay` is `false` for this response? even though this is the default it would add some clarity.